semantics/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "fe-semantics"
+version = "0.1.0"
+authors = ["Grant Wuerker <gwuerker@gmail.com>"]
```

Comment:
Can we make this `"Ethereum Foundation <snakecharmers@ethereum.org>"`. This also reminds me that I still need to double check why the license was changed (a long time ago) from MIT to  Apache 2.0. I think if we want to stick with Apache then at least we should dual license with MIT (that's what Rust does) because most of our projects are MIT licensed.